src/app/modules/workers/pages/workers-page/workers-page.component.ts

```diff
@@ -1,24 +1,68 @@
-import {Component, OnInit} from '@angular/core';
-import {EmployeeFiltersState} from '../../../../core/models/employee-filters-state';
-import {WorkersFacadeService} from './workers-facade.service';
+import { Component } from '@angular/core';
+import { WorkerFormComponent } from '../../components/worker-form/worker-form.component';
+import type { MatDialog } from '@angular/material/dialog';
+import type { OnInit } from '@angular/core';
+import type { Observable } from 'rxjs/internal/Observable';
+import type { ActivatedRoute, Router } from '@angular/router';
+import type { EmployeeFiltersState } from 'src/app/core/models/employee-filters-state';
+import type { WorkersFacadeService } from './workers-facade.service';
+import type { Employee } from 'src/app/core/models/employee';
 
 @Component({
   selector: 'app-workers-page',
   templateUrl: './workers-page.component.html',
-  styleUrls: ['./workers-page.component.scss']
+  styleUrls: ['./workers-page.component.scss'],
 })
 export class WorkersPageComponent implements OnInit {
-  public list$ = this.workersFacade.list$;
-  public filters$ = this.workersFacade.filters$;
+  public list$: Observable<Employee[]>;
+  public filters$: Observable<EmployeeFiltersState>;
 
-  constructor(private workersFacade: WorkersFacadeService) {
+  constructor(
+    private dialog: MatDialog,
+    private route: ActivatedRoute,
+    private router: Router,
+    private workersFacade: WorkersFacadeService,
+  ) {
+    this.list$ = this.workersFacade.list$;
+    this.filters$ = this.workersFacade.filters$;
```

Comment:
Nie ma potrzeby żeby to było w constructorze, mozna przpisac juz przy deklaracji tych zmiennych